src/Symfony/Component/Notifier/Bridge/Slack/Block/SlackSectionBlock.php

```diff
@@ -24,30 +24,46 @@ public function __construct()
     /**
      * @return $this
      */
-    public function text(string $text, bool $markdown = true): static
+    public function text(string $text, bool $markdown = true, bool $emoji = true, bool $verbatim = false): static
     {
         $this->options['text'] = [
             'type' => $markdown ? 'mrkdwn' : 'plain_text',
             'text' => $text,
         ];
 
+        // verbatim is only available for markdown
+        if ($markdown) {
+            $this->options['text']['verbatim'] = $verbatim;
+        } else {
+            $this->options['text']['emoji'] = $emoji;
+        }
```

Comment:
That's 3 correlated options, that makes it hard to reason about them.
Would it make sense to add only one option, either verbatim or emoji and send the appropriate setting to slack automatically?

Reply:
Well in term of behavior on Slack, `verbatim == !emoji)` (except [verbatim is for auto-URL parsing, emoji for emoji escaping](https://api.slack.com/reference/block-kit/composition-objects#text__fields)).
So maybe we can keep only `verbatim` option then having the following code if it's what you have in mind ? :)
```php
        if ($markdown) {
            $this->options['text']['verbatim'] = $verbatim;
        } else {
            $this->options['text']['emoji'] = !$verbatim
        }
```

But what about `SlackContextBlock` already having these two options and same condition ?
Removing `emoji` would be a BC (or we need to deprecate it ?), but keeping only `emoji` has less sense that `verbatim` in the meaning of the option in Slack :/